apps/store/src/lib/l10n/countries.ts

```diff
@@ -40,7 +40,7 @@ const localeCountries = Object.fromEntries(
 ) as Record<Locale, CountryLabel>
 
 export const getCountryByLocale = (locale: string): CountryData => {
-  const countryData = countries[localeCountries[locale as Locale]]
+  const countryData = countries[localeCountries[normalizeLocale(locale) as Locale]]
```

Comment:
Should this be 
```suggestion
  const countryData = countries[localeCountries[getLocale(locale)]]
```
then we don't need to cast it here?

Reply:
This may fall back to default locale which is something I'd prefer to avoid here

Reply:
Then I would prefer that we have `getLocaleWithFallback`/`getLocaleOrFallback` and `getLocale` 🤔